frontend/packages/component-library/package.json

```diff
@@ -59,6 +59,15 @@
       "require": "./dist/cjs/button/index.js"
     },
     "./button/index.css": "./dist/esm/button/index.css",
+    "./carousel": {
```

Comment:
super nit for the future: ideally, it's better to include import/export declarations in component-library pr

Reply:
Oh good call! I'll make a note of this on my next one.